Model/Resolver/Shipments.php

```diff
@@ -41,7 +41,7 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         foreach ($shipments as $shipment) {
             $orderShipments[] =
                 [
-                    'id' => base64_encode($shipment->getEntityId()),
+                    'id' => base64_encode((string) $shipment->getEntityId()),
```

Comment:
getEntityId() Int||null
base64_encode(string $string): string

Reply:
no whitespace after (string) would be standard, too